spring-boot-actuator/pom.xml

```diff
@@ -359,5 +359,11 @@
 			<artifactId>snakeyaml</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+		    <groupId>org.neo4j</groupId>
+		    <artifactId>neo4j-ogm-test</artifactId>
+		    <version>${neo4j-ogm.version}</version>
```

Comment:
Unfortunately we are not allowed to use this dependency due to its licensing (that's why we don't provide dependency management for it either).